src/api/handlers/categories.go

```diff
@@ -14,23 +14,25 @@ import (
 	"github.com/gin-gonic/gin"
 )
 
-type CreateCategoryRequest struct {
+type createCategoryRequest struct {
 	Name string `json:"name"`
 }
 
-type GetAllCategoriesResponse struct {
+type getAllCategoriesResponse struct {
 	Categories []categories.Category `json:"categories"`
 }
 
-type InsertCategoryResponse struct {
-	Id primitive.ObjectID `json:"id"`
+type insertCategoryResponse struct {
+	ID primitive.ObjectID `json:"id"`
```

Comment:
why rename?

Reply:
This type use only by the handler, therefore it shouldn't be a public